packages/react/src/auto/AutoTable.tsx

```diff
@@ -15,4 +15,6 @@ export type AutoTableProps<
   pageSize?: number;
   initialCursor?: string;
   initialDirection?: string;
+  sort?: { [key: string]: "Ascending" | "Descending" };
+  filter?: { [key: string]: { [key: string]: any } };
```

Comment:
Just found that you can also do this:
```ts
  sort?: Options["sort"];
  filter?: Options["filter"];
```

The `Options` is referenced from the function, and it will have a full type like this:
<img width="605" alt="CleanShot 2024-07-18 at 15 05 36@2x" src="https://github.com/user-attachments/assets/36b32acd-608d-4e23-aeb7-a5e05d1b2e42">